docs/en/concept/config.md

```diff
@@ -302,16 +311,16 @@ sink {
      source_table_name = "sql"
      username = "seatunnel=2.3.1"
      password = "$a^b%c.d~e0*9("
-  }
+    }
 }
 ```
 
-Some Notes:
-- Quota with `'` if the value has special character such as `(`
-- If the replacement variables is in `"` or `'`, like `resName` and `nameVal`, you need add `"`
-- The value can't have space `' '`, like `-i jobName='this is a job name' `, this will be replaced to `job.name = "this"`
-- If you want to use dynamic parameters, you can use the following format: -i date=$(date +"%Y%m%d").
-
+### Important Notes:
+- If a value contains special characters like `(`, enclose it in single quotes (`'`).
+- If the substitution variable contains double or single quotes (e.g., `"resName"` or `"nameVal"`), you need to include them with the value.
+- The value cannot contain spaces (`' '`). For example, `-i jobName='this is a job name'` will be replaced with `job.name = "this"`.
+- For dynamic parameters, you can use the following format: `-i date=$(date +"%Y%m%d")`.
+- 
```

Comment:
Please share system reserved keywords in here, link to `sink-options-placeholders.md`.